Resources/Prototypes/Floof/Traits/physical.yml

```
@@ -141,4 +145,26 @@
         Drink,
       ]
 
-
+- type: trait
+  id: Heavyweight
+  category: Physical
+  points: -2 # Has pros and cons, not sure
+  requirements:
+  - !type:CharacterJobRequirement
+    inverted: true
+    jobs:
+    - Borg
+    - MedicalBorg
+  - !type:CharacterSpeciesRequirement 
+    inverted: true
+    species:
+    - Harpy # Doesn't really make sense with their wings and flight ability
```

```suggestion
    - Harpy # Doesn't really make sense with their wings and flight ability
    - Diona # Trees are heavy as it is.
    - Chitinids # Ants got weight.
```
You may want to consider adding `Diona` and `Chitinid`, since they are both species that feature higher fixture density as part of their unique traits.

For reference, the `BaseMobSpeciesOrganic` has the fixture density set to `185`. Chitinids have this set to `220` instead, while Diona have a whopping `300` as their density.

Recall that both Rodentia and Felinids are currently restricted from the lightweight trait due to their already less than normal density. IPCs have the standard density. 

**Whether or not you want to restrict IPCs as well is up to your discretion.**

---

Makes sense to me! An IPC made out of denser materials seems reasonable to me, so I'm not going to add them currently. I'm actually a little surprised IPCs are standard density.